lib/middleware/user-data/user-data.js

```diff
@@ -36,12 +36,10 @@ const ENV = {
 const dataStoreType = userDataStoreClient.offline ? 'memory' : 'remote'
 
 const dataStoreMemory = {
-  loadData: async (args, logger, data = {}) => {
-    return Promise.resolve(data)
+  async loadData (args, logger, data = {}) {

```

Comment:
1. Async functions (which these were) do not need to explicitly return from `Promise.resolve()` because the return of an async function _is_ a Promise
2. The syntax `{ loadData: async (args, logger, data = {}) => {} }` is pointless, there's no benefit to using an arrow function here and the same result can be achieved with `{ async loadData (args, logger, data = {}) {} }`